chromium_src/chrome/browser/flag_descriptions.cc

```diff
@@ -26,4 +26,6 @@ const char kBraveAdblockCosmeticFilteringDescription[] =
     "Enable support for cosmetic filtering";
 const char kBraveSyncName[] = "Enable Brave Sync";
 const char kBraveSyncDescription[] = "Brave Sync is disabled by default";
+const char kBraveCustomHomepageName[] = "Enable Brave Custom Homepage";
```

Comment:
same as above re consistent terminology. It's very confusing when we refer to the same thing with different names

Reply:
fixed.